README.md

```diff
@@ -147,61 +164,35 @@ containerize.  It is recommended that most users use a version tag (e.g.
 |`cisagov/example:latest`| The most recent release image pushed to a container registry.  Pulling an image using the `:latest` tag [should be avoided.](https://vsupalov.com/docker-latest-tag/) |
 
 See the [tags tab](https://hub.docker.com/r/cisagov/example/tags) on Docker
-Hub for a list of all the supported tags.
+Hub for a list of all the supported tags. -->
 
 ## Volumes ##
 
 | Mount point | Purpose        |
 |-------------|----------------|
-| `/var/log`  |  Log storage   |
-
-## Ports ##
-
-The following ports are exposed by this container:
-
-| Port | Purpose        |
-|------|----------------|
-| 8080 | Example only; nothing is actually listening on the port |
-
-The sample [Docker composition](docker-compose.yml) publishes the
-exposed port at 8080.
+| `/home/cisa`  |  Log storage   |
```

Comment:
See my previous comment about whether or not this volume is truly needed.  If it is, please come up with a more accurate purpose description than "Log storage".  If it isn't, this section should be commented out and then add something like "There are no volumes for this container."

Reply:
Resolved in f8c135f56bb2837155838b04234e0119e9e95a08